src/codeu/chat/common/NetworkCode.java

```diff
@@ -35,5 +35,7 @@
       RELAY_READ_REQUEST = 27,
       RELAY_READ_RESPONSE = 28,
       RELAY_WRITE_REQUEST = 29,
-      RELAY_WRITE_RESPONSE = 30;
+      RELAY_WRITE_RESPONSE = 30,
+      SERVER_INFO_REQUEST = 33,
+      SERVER_INFO_RESPONSE = 34; 
```

Comment:
It would be a good idea to change `SERVER_INFO_REQUEST` and `SERVER_INFO_RESPONSE` to something more descriptive like `SERVER_VERSION_REQUEST`.